src/nbl/ui/CWindowX11.cpp

```diff
@@ -28,6 +31,9 @@ NBL_SYSTEM_DECLARE_DYNAMIC_FUNCTION_CALLER_CLASS(X11, system::DefaultFuncPtrLoad
     ,XGetErrorDatabaseText
     ,XGetErrorText
     ,XGetGeometry
+    ,XFindContext
+    ,XUniqueContext
+    ,XSaveContext
 );
```

Comment:
regarding the TODO, its the `ui` namespace (system is not for interfacing with the user)

Reply:
i already moved all the window classes to ui